README.md

```diff
@@ -80,7 +80,7 @@ The `csharp-libraries/NanoVer.Protocol` folder contains C# implementations of cl
 
 * Install Anaconda (avoid Anaconda 2.7 as it is outdated)
```

Comment:
This warning about 2.7 is probably irrelevant by now?

Reply:
Indeed, the 2.7 version does not seem to be on the as prominent anymore.